README.md

```diff
@@ -20,7 +20,7 @@ Todo:
 | `mermaid-export.outputWidth`  | 0       | Override the width of the output diagram (set to 0 for automatic sizing)           |
 | `mermaid-export.outputHeight` | 0       | Override the height of the output diagram (set to 0 for automatic sizing)          |
 | `mermaid-export.theme`        | default | Theme for output file (default, dark, forest or neutral)                           |
-| `mermaid-export.config`       |         | Extra configuration options file (see https://mermaidjs.github.io/mermaidAPI.html) |
+| `mermaid-export.config`       |         | Extra configuration options file (see https://mermaidjs.github.io/#/mermaidAPI) |
```

Comment:
As of now the link should be https://mermaid-js.github.io/mermaid/#/mermaidAPI.

Reply:
Thanks! Fixed in fdc02870099c2b9fe9c0083f7cdd4248e92cb481.